tracks/rust/exercises/clock/mentoring.md

```diff
@@ -0,0 +1,41 @@
+### Concepts
+
+- derive
+- struct
+- traits
+
+### Reasonable solutions
+
+A reasonable solution should do the following:
```

Comment:
This indicates that we are awaiting a lit of behaviors that a reasonable solution should do.   Seems to be missing some information.

Reply:
@kotp I think you have a valid point here.  This PR appears to be stale.  Should we go ahead and merge as is and let someone else pick this up later?

Reply:
Yeah, it will definitely be noticed, and if it isn't updated in the next couple of days (I will let the mentors know in the Rust room) we should merge it, it will draw the needed attention in that way.

Reply:
At that time perhaps we should add a blurb to the file that states `information needed here` or something along those lines.